libsmtutil/SolverInterface.h

```diff
@@ -17,7 +17,7 @@
 
 #pragma once
 
-#include <libsolidity/formal/Sorts.h>
+#include <libsmtutil/Sorts.h>
 
 #include <libsolidity/ast/Types.h>
```

Comment:
Is this needed in this file?

Reply:
Yea, `Expression` has a constructor based on `Type`